net-libs/libnma/libnma-1.10.2.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg vala
+
+DESCRIPTION="NetworkManager GUI library"
+HOMEPAGE="https://wiki.gnome.org/Projects/NetworkManager https://gitlab.gnome.org/GNOME/libnma"
+
+LICENSE="GPL-2+"
+SLOT="0"
+# pkcs11 default enabled as it's a small dep often already present by libnma users, and it was default enabled as IUSE=+gcr in nm-applet before
+IUSE="gtk-doc +introspection +pkcs11 vala"
+REQUIRED_USE="vala? ( introspection )"
+KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+DEPEND="
+	>=gui-libs/gtk-4.0:4
+	app-text/iso-codes
+	net-misc/mobile-broadband-provider-info
+	>=dev-libs/glib-2.38:2
+	>=x11-libs/gtk+-3.12:3[introspection?]
+	>=net-misc/networkmanager-1.7[introspection?]
+	pkcs11? ( >=app-crypt/gcr-3.14:0= )
+	introspection? ( >=dev-libs/gobject-introspection-1.56:= )
+"
+RDEPEND="${DEPEND}
+	!<gnome-extra/nm-applet-1.16.0" # gschema moved to here before nm-applet-1.16.0
+BDEPEND="
+	dev-libs/libxml2
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+	gtk-doc? ( dev-util/gtk-doc
+		app-text/docbook-xml-dtd:4.3 )
+	vala? ( $(vala_depend)
+		net-misc/networkmanager[vala]
+	)
+"
+
+src_prepare() {
+	default
+	use vala && vala_setup
+	xdg_environment_reset
+}
+
+src_configure() {
+	local emesonargs=(
+		-Dlibnma_gtk4=true
+		$(meson_use pkcs11 gcr)
+		-Dmore_asserts=0
+		-Dgcr_gtk4=false
```

Comment:
Should we enable this and depend on `gcr:4`. What do you think?

https://src.fedoraproject.org/rpms/libnma/blob/rawhide/f/libnma.spec#_14 looks like they're not enabling it...?

Reply:
I don't think we should. 
- upstream says this feature is experimental:
https://gitlab.gnome.org/GNOME/libnma/-/blob/main/meson_options.txt#L3
- I tried to build this with gcr-3.92.0:4 (with `-Dgtk4=true`) which is gcr4 iirc (at least telling from the files it installs) and it failse with gcr_gtk4 not found. It also says gcr_gtk3 not found, for whatever reason, but this doesn't seem to bother it. 

So we got some puzzle piece missing.